plugins/runtime/threads/thread-dump-modal.component.ts

```diff
@@ -0,0 +1,29 @@
+namespace Runtime {
+
+  export class ThreadDumpModalController {
+    resolve: { dumpedThreads: string };
+
+    get dumpedThreads() {
+      return this.resolve.dumpedThreads;
+    }
+  }
+
+  export const threadDumpModalComponent: angular.IComponentOptions = {
+    bindings: {
+      resolve: '<',
+      close: '&'
+    },
+    template: `
+    <div class="modal-header">
+    <button type="button" class="close" aria-label="Close" ng-click="$ctrl.close()">
+        <span class="pficon pficon-close" aria-hidden="true"></span>
+    </button>
+    <h4 class="modal-title">Thread Dump</h4>
+    </div>
+    <code>
```

Comment:
modal body should be wrapped with `<div class="modal-body">`. see
https://www.patternfly.org/v3/pattern-library/forms-and-controls/modal-overlay/#code